CC_ranking.py

```diff
@@ -0,0 +1,29 @@
+# Importing necessary modules
+from urllib.request import Request, urlopen   # to access data from the url 
+import urllib,json                            # to parse json data
+import pandas as pd                           # to organise data in csv format
+
+ContestID = 'JUNE19A'
+#path = 'https://www.codechef.com/api/rankings/'+ContestID
+path = 'https://www.codechef.com/api/rankings/'+ContestID+'?sortBy=score&order=desc&page=1&itemsPerPage=100&fbclid=IwAR08gwUo_Opd6La9ha1OWvVdepRGpUeZK-bBxTc9tCjNL0G6MXEQEgjk4-Q'
```

Comment:
```suggestion
path = 'https://www.codechef.com/api/rankings/'+ContestID+'?sortBy=score&order=desc&page=1&itemsPerPage=100'
```

Reply:
Good start, but this only fetches the first page of the ranklist. Need to also add Page as a parameter in `path`, and loop for all pages.